core/contracts/oracle/implementation/Governor.sol

```diff
@@ -5,7 +5,7 @@ pragma experimental ABIEncoderV2;
 import "../../common/MultiRole.sol";
 import "../../common/FixedPoint.sol";
 import "../../common/Testable.sol";
-import "../../Finder.sol";
+import "../interfaces/FinderInterface.sol";
 import "../interfaces/IdentifierWhitelistInterface.sol";
 import "./Voting.sol";
```

Comment:
Same question as above about Voting vs VotingInterface

Reply:
This one can probably just using VotingInterface. But I think we should change in the same PR that we change the one above.